python/graphstorm/wholegraph/wholegraph.py

```diff
@@ -437,17 +450,19 @@ def save_to_file(
         -------
         None
         """
+        assert self._tensor is not None, "Please create WholeGraph tensor first."
```

Comment:
```suggestion
        assert self._tensor is not None, "Please create WholeGraph tensor first by either create WholeGraphDistTensor with use_wg_optimizer=False or by calling attach_wg_optimizer after WholeGraphDistTensor is initialized."
```
As the behavior of WholeGraphDistTensor is different from other DistTensors, we need to be more specific in the error message.